user/docs/index.md

```diff
@@ -2,6 +2,101 @@
+
+The guide mainly aims to help you install NeoForge using the official Minecraft launcher provided by Mojang. There are also a number of third-party launchers that mostly automate this process for you, some of which are mentioned in the article on [third-party launchers][launchers].
+
+## Java
+
+In order to run NeoForge, you will first need to have Java installed on your computer. Java is the programming language Minecraft and NeoForge are written in. While Minecraft uses the launcher to download the required Java version, NeoForge requires you to install Java yourself.
+
+The required Java version differs depending on what Minecraft version you want to run:
+
+| Minecraft version | Java version |
+|:-----------------:|:------------:|
+|   1.20.2-1.20.4   |      17      |
+|   1.20.5-latest   |      21      |
+
+:::info
+While NeoForge exists for Minecraft **1.20.1**, we recommend using Forge on that version instead, since it had longer support for Minecraft 1.20.1. We only recommend using NeoForge on Minecraft 1.20.2 and newer.
+:::
+
+### Testing For Java
+
+_If you are sure that you do not have Java installed, you can skip to [Installing Java][installingjava]._
+
+In many cases, you may already have Java installed on your system. As such, you need to verify if the version is correct.
+
+- Open a terminal. The way to do this depends on the operating system you are running:
+
+<Tabs defaultValue="windows">
+  <TabItem value="windows" label="Windows">
+In the Start Menu at the bottom left, search for `Command Prompt` and press Enter.
+  </TabItem>
+  <TabItem value="macos" label="MacOS">
+Open Finder. In Finder, open the Applications/Utilities folder and double-click Terminal.
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+Open the terminal for your Linux distribution. Common names would be `GNOME Terminal` or `Konsole`, however it may vary depending on your exact setup.
+  </TabItem>
+</Tabs>
+
+- Type the following command: `java -version` and press Enter.
+- If an error is displayed, then Java is not installed, and you can skip to the [Installing Java][installingjava] section.
+- If Java is installed, you should see the following output (or something similar):
+```
+openjdk version "21.0.4" 2024-07-16 LTS
+OpenJDK Runtime Environment Temurin-21.0.4+7 (build 21.0.4+7-LTS)
+OpenJDK 64-Bit Server VM Temurin-21.0.4+7 (build 21.0.4+7-LTS, mixed mode, sharing)
+```
+- Verify that the first number after the `version` bit matches the Java version needed for the desired Minecraft version.
+  - E.g., `openjdk version "21.0.4" 2024-07-16 LTS` is a Java 21 version and as such fit for Minecraft 1.20.5 and newer.
+  - If the version does not match, then you will have to [install the correct Java version][installingjava].
+- If everything went smoothly, continue to [Installing NeoForge][installingneoforge].
+
+### Installing Java
+
+The way to install Java depends on your operating system. Always make sure you're grabbing the correct version!
```

Comment:
Probably should mention 64-bit specifically as 32-bit installations still exist.